source/application/AMDatamanAppController.cpp

```diff
@@ -1875,3 +1932,33 @@ void AMDatamanAppController::onOpenOtherDatabaseClicked()
 		connect(newScanDataView, SIGNAL(launchScanConfigurationsFromDb(QList<QUrl>)), this, SLOT(onLaunchScanConfigurationsFromDb(QList<QUrl>)));
 	}
 }
+
+void AMDatamanAppController::timerEvent(QTimerEvent *)
+{
+	updateStorageProgressBar();
+}
+
+void AMDatamanAppController::updateStorageProgressBar()
+{
+	if(usingLocalStorage() && storageInfo_.isValid()) {
+
+		storageInfo_.refresh();
+		double storageUsed = double(storageInfo_.bytesTotal() - storageInfo_.bytesAvailable());
+		double percentageUsed = (storageUsed / storageInfo_.bytesTotal()) * 100;
+
+		if(percentageUsed > 85) {
+
+			if(!storageWarningProvided_) {
+				double percentageRemaining = 100 - percentageUsed;
+				AMErrorMon::alert(this, AMDATAMANAPPCONTROLLER_LOCAL_STORAGE_RUNNING_LOW, QString("Warning: Local storage space is at %1%. Please inform the beamline staff.").arg(percentageRemaining));
+				storageWarningProvided_ = true;
```

Comment:
How will we clear the warning provided flag? By far, it seems that the only way to clear `storageWarningProvided_` is via:
 -- close acquaman
 -- remove/delete files on local storage to gain free space ? 10%
 -- restart acquaman